src/types/utils.cpp

```diff
@@ -4,6 +4,10 @@
 #include "precomp.h"
 #include "inc/utils.hpp"
 #include <Objbase.h>
+
+#include <bcrypt.h>
+#include <wil/resource.h> // reinclude wil/resource to get the bcrypt defines.

```

Comment:
Maybe objbase, bcrypt should be in types/precomp.h?

Reply:
probs

Reply:
though there's a weird thing about wil. I want to keep this targeted, and including bcrypt in precomp will require us to move the wil include out of LibraryIncludes, because wil only adds the bcrypt types if it is included after bcrypt.

Right now, it's:

```
+precomp.h
|`+LibraryIncludes
| |-stl
| |-gsl
| `-wil
`+windows.h
`+(bcrypt.h?)
`+(wil again? nah)
```

Reply:
Or to move the LibraryIncludes.h down further in the types/precomp.h. Which is the case in some of the other precomp.hs around this project for this reason.

Reply:
done/done